data/ideas.json

```diff
@@ -27,12 +27,100 @@
         "status": "rejected"
     },
     {
-        "id": "a6bbff48-113a-46fb-9338-9e5844275333",
+        "id": "6055e10e-1b89-4213-81cf-360c2e6dc6a1",
         "picture": "./img/default_idea.png",
-        "header": "Simply Amazing HEADER 1",
-        "description": "Yet another interesting text. Just read it",
-        "askedSum": 300,
-        "userId": "57226ce8-3e44-4f98-83b1-eff8e257a842",
+        "header": "New liver, for Alcoholics",
+        "description": "Perhaps it you need one?",
+        "askedSum": 1500,
+        "userId": "7a693f37-ccb4-4ecb-be2b-8e34749430b5",
+        "status": "pending"
+    },
+    {
+        "id": "6ae1ea20-6e79-45f8-8265-eba8a3d22677",
+        "picture": "./img/default_idea.png",
+        "header": "Epic Star Wars Header",
+        "description": "Epic description, that will astonish you!",
+        "askedSum": 500,
+        "userId": "d6e15668-dcc1-49e6-9343-4972d100f329",
+        "status": "accepted"
+    },
+    {
+        "id": "99566354-2870-4003-b399-bd7e65eeed9a",
+        "picture": "./img/default_idea.png",
+        "header": "Headering header",
+        "description": "Descriptive description",
+        "askedSum": 750,
+        "userId": "26877c11-30ac-4f93-8a89-8136c3ddb2e4",
+        "status": "rejected",
+        "donations": [
```

Comment:
Ideas should not store donations - donations are stored separately

Reply:
fixed